config/locales/pt_br.yml

```
@@ -0,0 +1,243 @@
+---
+pt_br:
```

Just as a small note, some apps might use `pt-BR` instead of `pt_BR`. Not sure if or how this can be handled though.

```suggestion
pt-br:
```

---

Thanks for flagging this! From a code search, it looks like it may not be strictly necessary: https://github.com/search?type=code&q=path%3Aconfig%2Flocales%2F*.yml+%2F%5Ept%5B_-%5Dbr%2F


And seems like both RubyGems.org and Mastodon use `pt-BR` so I'll go with that. I'll make that change.